### PR TITLE
17193 Style wp attributes based on latest mockups

### PR DESCRIPTION
**Pull request by myabc**
This does not include any inline-editing related styles.

https://community.openproject.org/work_packages/17193

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://feature-17193-work-package-attribute-styling.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/stylesheets/content/work_package_details/_overview_tab.sass [hunk 1]
```diff
@@ -32,17 +32,21 @@
...
+  font-size: 14px
   float: left
-  margin: 15px 0 0 0
+  margin: 3px 0 0 0
   width: 100%
 
-  dl
-    margin: 0 0 24px 0
+  dd, dt
+    margin: 0 0 10px 0
```
Comment (ulferts) [inline app/assets/stylesheets/content/work_package_details/_overview_tab.sass:49]: @myabc what about the space between the various groups? They seem to be rather narrow.

**Comment by ulferts** [comment]
@myabc I am still unsure about the scope of this PR. When comparing this to http://clients.zzmedia.net/openproject/inplace-edit/styleguide/ I see quite some differences even in places I would not deem to be related to inplace editing.

**Comment by myabc** [comment]
@ulferts this PR contains only a subset of the changes that we need to make to consider # 17193 "complete". I've also added a comment to the Use Story on why the "style guide", in its current form, is problematic.

**Comment by ulferts** [comment]
@myabc seen that update. Although I think that some of the margins could be changed here I am OK to merge as this does not make anything worse.

**Comment by myabc** [comment]
@ulferts ok, sounds good.

**Pull request merged**